doc/classes/Texture.xml

```diff
@@ -117,22 +117,25 @@
 	</methods>
 	<constants>
 		<constant name="FLAG_MIPMAPS" value="1" enum="Flags">
-			Generate mipmaps, to enable smooth zooming out of the texture.
+			Generates mipmaps, generates smaller versions of the same texture to use when zoomed out, keeping the ratio. 
 		</constant>
 		<constant name="FLAG_REPEAT" value="2" enum="Flags">
-			Repeat (instead of clamp to edge).
+			Repeats texture (instead of clamp to edge).
 		</constant>
 		<constant name="FLAG_FILTER" value="4" enum="Flags">
-			Turn on magnifying filter, to enable smooth zooming in of the texture.
+			Magnifying filter, to enable smooth zooming in of the texture.
 		</constant>
 		<constant name="FLAGS_DEFAULT" value="7" enum="Flags">
 			Default flags. Generate mipmaps, repeat, and filter are enabled.
 		</constant>
 		<constant name="FLAG_ANISOTROPIC_FILTER" value="8" enum="Flags">
+			Generate anisotropic mipmap filtering, generates smaller versions of the same texture with different aspect ratios, more effective on planes often shown going to the horrizon.
```

Comment:
Inconsistent with other descriptions (tense and style):

Anisotropic mipmap filtering.  Generates smaller versions of the same texture with different aspect ratios.

> more effective on planes often shown going to the horrizon.

I'm not clear what this means.